optuna/storages/_rdb/alembic/env.py

```diff
@@ -5,7 +5,7 @@
 from sqlalchemy import engine_from_config
 from sqlalchemy import pool
 
-import optuna
+import optuna.storages._rdb.models
```

Comment:
This change was required to avoid the following import error.

```console
$ alembic revision --autogenerate  --rev-id v3.0.0.b
Traceback (most recent call last):
  File "/Users/yanase/pfn/code/optuna/.venv/bin/alembic", line 8, in <module>
    sys.exit(main())
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/config.py", line 588, in main
    CommandLine(prog=prog).main(argv=argv)
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/config.py", line 582, in main
    self.run_cmd(cfg, options)
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/config.py", line 559, in run_cmd
    fn(
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/command.py", line 227, in revision
    script_directory.run_env()
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/script/base.py", line 563, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/util/pyfiles.py", line 92, in load_python_file
    module = load_module_py(module_id, path)
  File "/Users/yanase/pfn/code/optuna/.venv/lib/python3.9/site-packages/alembic/util/pyfiles.py", line 108, in load_module_py
    spec.loader.exec_module(module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "alembic/env.py", line 25, in <module>
    target_metadata = optuna.storages._rdb.models.BaseModel.metadata
AttributeError: module 'optuna.storages._rdb' has no attribute 'models'
```